snapshot/process.go

```diff
@@ -57,6 +57,12 @@ func chooseProcessor(ctx context.Context, logger *logrus.Entry,
 		}
 	}
 
+	proxyhandler := func() (bool, []mount.Mount, error) {
```

Comment:
Better be `proxyhandler` -> `proxyHandler`.

Reply:
Fixed.